docassemble/MACourts/data/sources/district_courts.json

```
@@ -217,6 +295,13 @@
       "state": "MA",
       "address": "305 Walden St."
     },
+    "ADA Coordinators": [
+      {
+        "name": "Evan Gilman",
+        "phone": "(978) 369-0500 ext. 623",
+        "email": "evan.gilman@jud.state.ma.us
```

```suggestion
        "email": "evan.gilman@jud.state.ma.us"
```
This just needs a quote character at the end.